client/qiskit_serverless/core/clients/serverless_client.py

```diff
@@ -280,7 +280,7 @@ def status(self, job_id: str):
         default_status = "Unknown"
         response_data = safe_json_request_as_dict(
             request=lambda: requests.get(
-                f"{self.host}/api/{self.version}/jobs/{job_id}/",
+                f"{self.host}/api/{self.version}/jobs/{job_id}?with_result=false",
```

Comment:
This is a way to do it but `requests` offers us a better way to manage the params:
https://github.com/Qiskit/qiskit-serverless/blob/main/client/qiskit_serverless/core/files.py#L141

Reply:
Done!